src/io.jl

```diff
@@ -217,41 +131,41 @@ function create_internal_structures(table_tree::TableTree, connection)
     for (u, v) in MetaGraphsNext.edge_labels(graph)
         compute_flows_partitions!(
             graph[u, v].rep_periods_partitions,
-            table_tree.partitions.flows,
+            TulipaIO.get_table(connection, "flows_rep_periods_partitions"),
             u,
             v,
             representative_periods,
         )
     end
 
-    # For timeframe, only the assets where is_seasonal is true are selected
-    for row in TulipaIO.get_table(Val(:raw), connection, "assets_data")
-        if row.is_seasonal
-            # Search for this row in the TulipaIO.get_table(Val(:raw), connection, "assets_data") and error if it is not found
-            found = false
-            for partition_row in eachrow(table_tree.partitions.assets[:timeframe])
-                if row.name == partition_row.asset
-                    graph[row.name].timeframe_partitions = _parse_rp_partition(
-                        Val(Symbol(partition_row.specification)),
-                        partition_row.partition,
-                        1:timeframe.num_periods,
-                    )
-                    found = true
-                    break
-                end
-            end
-            if !found
-                graph[row.name].timeframe_partitions =
-                    _parse_rp_partition(Val(:uniform), "1", 1:timeframe.num_periods)
-            end
-        end
```

Comment:
I think we discussed rewriting this with TulipaIO later, right @suvayu?
I'll leave this comment so that we create an issue for it.

Reply:
Yes, this and the `groupby` & `filter`s below.